packages/nextra/src/loader.ts

```diff
@@ -256,12 +258,20 @@ ${finalResult}
 
 setupNextraPage({
   pageNextRoute: ${JSON.stringify(pageNextRoute)},
-  pageOpts: ${JSON.stringify(pageOpts)},
+  pageOpts: ${stringifiedPageOpts},
   nextraLayout: __nextra_layout,
   themeConfig: ${themeConfigImport ? '__nextra_themeConfig' : 'null'},
   Content: MDXContent,
   hot: module.hot,
-  dynamicMetaItems: ${JSON.stringify(dynamicMetaItems)}
+  pageOptsChecksum: ${JSON.stringify(hashFnv32a(stringifiedPageOpts))},
```

Comment:
anyway, `module.hot` code is no longer included and tree shaked due `process.env.NODE_ENV !== 'production'` check